src/components/popup-utils.js

```diff
@@ -32,6 +32,20 @@ export const useDynamicPosition = selectors => {
   return dimensions
 }
 
+export const useWindowDimensions = () => {
```

Comment:
Reading `window.innerWidth` / `window.innerHeight` potentially causes reflow, so this hook minimizes reading those properties.

https://gist.github.com/paulirish/5d52fb081b3570c81e3a